next-tavla/src/Board/scenarios/Table/components/TableHeader/styles.module.css

```diff
@@ -9,3 +11,10 @@
     font-weight: 600;
     margin: 0;
 }
+
+.duration {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    color: var(--main-text-color);
+}
```

Comment:
Jeg lurer egentlig litt på om vi burde lage utils klasser for farger også. Du kan la denne være foreløpig